string    active_controller
string[]  available_controllers

string    active_tracker
string[]  available_trackers

float32   thrust    # [%]
